### ISSUE TITLE
Blog post cannot be opened when localization is enabled in features. -RC2

### ISSUE THREAD

**Issue by zhubenben** [OrchardCMS/OrchardCore#8934]
Blog post cannot be opened when localization is enabled in features. -RC2

![2021-03-21_100100](https://user-images.githubusercontent.com/39297211/111891360-71107f00-8a2d-11eb-9cd7-65a513a8cb89.png)

**Comment by jtkech** [comment]
Yes, it's an encoding issue that can be bypassed by the way that some `IHtmlContent` use the encoder, here the Fluid `StringValue`. Related to #8887 where i commented

> Ah Okay, so you are using the master branch (nuget feed)

> If you can repro locally, would be good to re-try it with the last dev branch (cloudsmith feed)

The `StringValue` has been updated in Fluid, we also updated another custom `IHtmlContent` in OrchardCore

So would be good to try it with the last dev branch from the cloudsmith feed

**Comment by Piedone** [comment]
So, this should now be fixed.
